infrastructure/src/main/java/org/corfudb/infrastructure/IDataStore.java

```diff
@@ -11,32 +14,62 @@
  * <p>Created by mdhawan on 7/27/16.
  */
 public interface IDataStore {
+
+    @Deprecated
```

Comment:
Please avoid adding interface which is already deprecated.

Reply:
These methods used before, I made them deprecated but in the PR it looks as a just created them.  In reality, I added some new methods and deprecated these ones

Reply:
yeah it looks like that. ok then.